clients/src/main/java/org/apache/kafka/clients/admin/QuorumInfo.java

```diff
@@ -174,4 +194,22 @@ public String toString() {
                 ')';
         }
     }
+
+    public static class Node {
+        private int nodeId;
+        private List<Endpoint> endpoints;
```

Comment:
Use `RaftVoterEndpoint` here, not `Endpoint`.

`Endpoint` shouldn't be used in admin APIs.

Reply:
Fixed